test/utils/mnemonic.js

```diff
@@ -0,0 +1,26 @@
+/*
+ * LiskHQ/lisk-commander
+ * Copyright © 2017–2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+import * as mnemonic from '../../src/utils/mnemonic';
+
+describe('mnemonic utils', () => {
+	it('createMnemonicPassphrase should be a function', () => {
+		return expect(mnemonic.createMnemonicPassphrase).to.be.a('function');
+	});
+
+	it('isValidMnemonicPassphrase should be a function', () => {
+		return expect(mnemonic.isValidMnemonicPassphrase).to.be.a('function');
+	});
```

Comment:
Previously we had a sanity-check integration test to confirm that a mnemonic passphrase created with our exported method counted as valid according to the other exported method. We don't want to test third-party code, but this test would still give us confidence we were exporting things as intended.